docs/shared/guides/dynamic-mfe-angular.md

```diff
@@ -0,0 +1,167 @@
+# Advanced Angular Micro Frontends with Dynamic Module Federation
+
+Dynamic Module Federation is a technique that allows an application to determine the location of its remote applications at runtime. It helps to achieve the use case of **“Build once, deploy everywhere”**.
+
+“Build once, deploy everywhere” is the concept of being able to create a single build artefact of your application and deploy it to multiple environments such as staging and production.
+
+The difficulty in achieving this with a Micro Frontend Architecture using Static Module Federation is that our Remote applications will have a different location (or URL) in each environment. Previously, to account for this, we would have had to specify the deployed location of the Remote applications and rebuild the application for the target environment.
+
+This guide will walk through how the concept of “Build once, deploy everywhere” can be easily achieved in a Micro Frontend Architecture that uses Dynamic Module Federation.
+
+## Prerequisites
+
+This guide follows the previous Angular Micro Frontend guide. It will use the same workspace that was created in that guide.
+If you have not completed it, you can find that guide [here](/guides/setup-mfe-with-angular).
```

Comment:
nit: since a single line break doesn't actually break the line in the rendered docs, I think it would be easier to reason about if we don't break the line here (as well as other places in this doc with a single line break) and only do it when we actually want to break lines. Wdyt?

Reply:
yeah it would be best. 2 spaces also breaks a line in MD. which sometimes GH doesn't render well. However, in this case, it is a single line break 😅